### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by Dsm0** [frankiezafe/gdosc#2]
I'm pretty new to godot, and I have no idea how to install this addon.

Could you add a basic tutorial for users like me?

**Comment by TimGremalm** [comment]
I'm in the same situation, I've no idea what to do. I've tried to place the output from the releasebuild under the projects addon-directory without any success.

**Comment by alexjsmac** [comment]
Yes same here, some additional guidance would be so appreciated, thanks!
